### PR TITLE
Refactory: Workflow Activity based on new notification queue

### PR DESCRIPTION
**Pull request by yamelsenih**
- Remove old implementation of send mail
- Add new queue notification

Related to: #3552

### CODE REVIEW

File: base/src/org/compiere/wf/MWFActivity.java [hunk 1]
```diff
@@ -1213,30 +1212,23 @@ else if (!(doc.processIt (DocAction.ACTION_Approve)))
...
-							+ ": " + doc.getDocumentNo(), 
-							(doc.getSummary() != null ? doc.getSummary() + Env.NL : "" )
+				//	Get instance for notifier
+				DefaultNotifier notifier = (DefaultNotifier) QueueLoader.getInstance().getQueueManager(DefaultNotifier.QUEUETYPE_DefaultNotifier)
+						.withContext(Env.getCtx())
+						.withTransactionName(trx.getTrxName());
+				//	Send notification to queue
+				notifier
+					.clearMessage()
+					.withApplicationType(DefaultNotifier.DefaultNotificationType_UserDefined)
```
Comment (rcapecce) [inline base/src/org/compiere/wf/MWFActivity.java:1222]: Hi @yamelsenih, in this case, when you use the "withApplicationTYpe" with the option "DefaultNotificationType_UserDefined", you don't have the user stablished.

**Review by rcapecce — CHANGES_REQUESTED** [review]
Hi @yamelsenih 
When you run this configuration with the DefaultNotificationType_UserDefined, you don't have the user in this line
https://github.com/erpcya/adempiere/blob/7caaa9692e8e858fce3c6582ae91b6228a31fed3/base/src/org/spin/queue/notification/DefaultNotifier.java#L313
Can you check this?

File: base/src/org/compiere/wf/MWFActivity.java (hunk 1 above)
Comment (SusanneCalderon) [reply base/src/org/compiere/wf/MWFActivity.java:1222]: Hi @yamelsenih 
When you do not specify an email address (Text in Workflow Node) you get an error.

**Comment by e-Evolution** [comment]
@yamelsenih can please solve the issue report to @rcapecce and @SusanneCalderon , this way I can merge this change

File: base/src/org/compiere/wf/MWFActivity.java (hunk 1 above)
Comment (yamelsenih) [reply base/src/org/compiere/wf/MWFActivity.java:1222]: Hi @all the user defined just allows define directly from user notification type and a notification can be send by email, telegram and other media type depending of user defined and supported client

**Comment by yamelsenih** [comment]
Hi @e-Evolution this was resolved yet

**Pull request merged**